airflow/providers/microsoft/azure/provider.yaml

```diff
@@ -30,21 +30,26 @@ integrations:
     tags: [azure]
   - integration-name: Microsoft Azure Blob Storage
     external-doc-url: https://azure.microsoft.com/en-us/services/storage/blobs/
+    logo: /integration-logos/azure/Blob Storage.svg
     tags: [azure]
   - integration-name: Microsoft Azure Container Instances
     external-doc-url: https://azure.microsoft.com/en-us/services/container-instances/
+    logo: /integration-logos/azure/Container Instances.svg
     tags: [azure]
   - integration-name: Microsoft Azure Cosmos DB
     external-doc-url: https://azure.microsoft.com/en-us/services/cosmos-db/
+    logo: /integration-logos/azure/Azure Cosmos DB.svg
     tags: [azure]
   - integration-name: Microsoft Azure Data Explorer
     external-doc-url: https://azure.microsoft.com/en-us/services/data-explorer/
     tags: [azure]
   - integration-name: Microsoft Azure Data Lake Storage
     external-doc-url: https://azure.microsoft.com/en-us/services/storage/data-lake-storage/
+    logo: /integration-logos/azure/Data Lake Storage.svg
     tags: [azure]
   - integration-name: Microsoft Azure Files
     external-doc-url: https://azure.microsoft.com/en-us/services/storage/files/
+    logo: /integration-logos/azure/Azure Files.svg
```

Comment:
Where will these files be saved? Can you automatically check if these files exist? 

Related file: https://github.com/apache/airflow/blob/master/scripts/ci/pre_commit/pre_commit_check_provider_yaml_files.py

Reply:
Those files are today in the [`airflow-site` repo at landing-pages/site/static/integration-logos](https://github.com/apache/airflow-site/tree/master/landing-pages/site/static/integration-logos)

I don't know if we can check that the files exist in **other repo** as a pre commit hook, I mean I can but it will fail in CI I guess because `airflow-site` won't be there at that time. 

So do you want to copy the logos into this repo instead, and then let this script generate:

    ../airflow-site/landing-pages/site/data/integrations.json
    ../airflow-site/landing-pages/site/static/integrations.json
     ../airflow-site/landing-pages/site/static/integrations-logos/**

Reply:
I guess the best idea would be to copy these files into this repository to the `/docs/integration-logos/` directory.

Reply:
Ok, I'll do that 
* copy the logos into `/docs/integration-logos`
* check in the pre commit hook that the logo file  referenced in each `provider.yaml` actually exist
* generate the `integrations.json` file directly into `$AIRFLOW_SITE_DIRECTORY/landing-pages/site/{data,static}/integrations.json`
* copy files over from `/docs/integration-logos` into `$AIRFLOW_SITE_DIRECTORY/landing-pages/site/static/integration-logos`

Reply:
SGTM

Reply:
done